client/app/_components/layout/header/Header.tsx

```diff
@@ -16,41 +15,42 @@ const Header = ({ hasNavigation = true }: HeaderProps) => {
             priority={true}
             src={`/images/logo/logo.png`}
             alt={'logo'}
-            width={'150'}
-            height={'150'}
-            className='h-auto w-auto'
+            //  Set the exact ratio of the original image so that you can avoid getting a warn
```

Comment:
Thanks, you can delete this comment later!

Reply:
https://github.com/HAK2024/fairy-share/pull/71/commits/3d2546c009f6cb80f5d1a9b4eef5815d2280c4ed